bitnami/grafana/templates/NOTES.txt

```diff
@@ -31,3 +31,15 @@ APP VERSION: {{ .Chart.AppVersion }}
 {{- include "common.warnings.rollingTag" .Values.imageRenderer.image }}
 {{- include "common.warnings.rollingTag" .Values.volumePermissions.image }}
 {{ include "grafana.validateValues" . }}
+
+{{/* Validate values of Grafana - Requirements to use an external database */}}
+{{- define "grafana.validateValues.database" -}}
+{{- $replicaCount := int .Values.grafana.replicaCount }}
+{{- if gt $replicaCount 1 -}}
+Grafana: replicaCount
+Using more than one replica requires using an external database to share data between Grafana instances.
+By default Grafana uses an internal sqlite3 per instance but you can configure an external MySQL or PostgreSQL.
+Please, ensure you provide a configuration file configuring the external database to share data between replicas.
+{{- end -}}
+{{- end -}}
```

Comment:
It would be great to keep this validation at the `_helpers.tpl` file with the rest of validation functions. We then can call it here (as you are doing) later.

Reply:
So leave the functions in `_helpers.tpl`, get rid of the append $messages which references it, and call the function in the notes?